Gruntfile.js

```diff
@@ -90,10 +91,16 @@ module.exports = function(grunt) {
 		}
 	});
 
-	grunt.loadTasks('tasks');
-	grunt.loadNpmTasks('grunt-contrib-clean');
-	grunt.loadNpmTasks('grunt-contrib-watch');
-	grunt.loadNpmTasks('grunt-simple-mocha');
+	if (process.env.NODE_ENV === 'production') {
+		require('./tasks/buildsources')(grunt);
+		require('./tasks/updatelibrary')(grunt);
+		grunt.loadNpmTasks('grunt-contrib-clean');
+	} else {
+		grunt.loadTasks('tasks');
+		grunt.loadNpmTasks('grunt-contrib-clean');
+		grunt.loadNpmTasks('grunt-contrib-watch');
+		grunt.loadNpmTasks('grunt-simple-mocha');
+	}
```

Comment:
In an attempt to reduce the amount of dependencies installed inside the Docker image, the `buildsources`,`updatelibrary` and `clean` tasks are required to install and build our polyfills. All the other grunt tasks are only required for development tasks.